astropy/units/core.py

```diff
@@ -1512,46 +1512,41 @@ def to_system(self, system):
         Returns
         -------
         units : list of `CompositeUnit`
-            The list is ranked so that units containing only the base
-            units of that system will appear first.
-        """
-        bases = set(system.bases)
-
-        def score(compose):
-            # In case that compose._bases has no elements we return
-            # 'np.inf' as 'score value'.  It does not really matter which
-            # number we would return. This case occurs for instance for
-            # dimensionless quantities:
-            compose_bases = compose.bases
-            if len(compose_bases) == 0:
-                return np.inf
-            else:
-                sum = 0
-                for base in compose_bases:
-                    if base in bases:
-                        sum += 1
+            The units are ordered by simplicity, preferring base units.
```

Comment:
This method does not prefer base units.

Reply:
It actually does - `u.gal` becomes `cm/s2`; `u.sr` becomes `rad2`. It is just that generally `.compose()` only provides composite units that sorted by shortness, removing those with 2 or more bases above the unit with the least bases.

Reply:
If the rule is that `to_system` returns the shortest version it can find, unless the unit can be expressed as a combination of exactly two base units, then that's what the docstring should say.

Reply:
That presumes I completely understand `.compose()` - this is why initially I just referred to it... It has a long list of sorts...

Reply:
If describing how the results are sorted is too complicated then perhaps it's best not to describe the sorting at all.

Reply:
I agree with @eerovaher here. Also I find "ordered by simplicity" to be fundamentally ambiguous: is it sorted by *increasing* or *decreasing* simplicity ?

Reply:
Actually, the examples explain it somewhat, so how about just referring to that? I.e.,
"With an attempt to sort simpler units to the start (see examples)."